Modules/ITS/itsQCTrending.json

```diff
@@ -0,0 +1,719 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "42",
+        "type": "2"
+      },
+      "monitoring": {
+        "url": "infologger:///debug?qc"
+      },
+      "consul": {
+        "url": "http://consul-test.cern.ch:8500"
+      },
+      "conditionDB": {
+        "url": "ccdb-test.cern.ch:8080"
+      }
+    },
+    "postprocessing": {
+      "ExampleTrendingThreshold": {
+        "active": "true",
+        "className": "o2::quality_control::postprocessing::TrendingTaskITS",
+        "moduleName": "QualityControl",
+        "detectorName": "ITS",
+        "dataSources": [
+          {
+            "types": ["repository","repository"],
+            "paths": ["qc/ITS/ITSTHRTask0/Threshold/Layer0/Stave0/Threshold","qc/ITS/ITSTHRTask0/DeadPixel/Layer0/Stave0/DeadPixelHITMAP"],
+            "names": [ "Threshold_l0_0", "DeadPixels_l0_0" ],
+            "reductorNames": ["o2::quality_control_modules::common::TH1Reductor", "o2::quality_control_modules::common::THnSparseReductor"],
+            "moduleNames": ["QcCommon","QcCommon"]
+          },
```

Comment:
Hi, I will have a closer look in August, now I only had a glimpse. Weren't you able to specify one reductor and only multiple names? For example TPC did that here https://github.com/AliceO2Group/QualityControl/blob/7ceb400fa2fc2374dfc43d6073838e51416fbf0e/Modules/TPC/run/tpcQCSimpleTrending.json#L35

So in your case it would be
```
          {
            "type": "repository",
            "path": "qc/ITS/ITSTHRTask0/Threshold/Layer0/Stave0/Threshold",
            "names": [ "Layer0/Stave0/Threshold/Threshold_l0_0",
                     "Layer0/Stave0/Threshold/Threshold_l0_1", ... ,
                     "Layer0/Stave1/Threshold/Threshold_l0_0", ...,
                     "Layer1/Stave4/Threshold/Threshold_l0_0" ], (basically all Threshold histograms listed here)
            "reductorName": "o2::quality_control_modules::common::TH1Reductor"
            "moduleName": "QcCommon",
          },
```
and similarly the second one for DeadPixelHITMAPs

Reply:
@iravasen I let you comment but it could go to another PR if we want to merge this one.

Reply:
Hi @knopers8, thanks for your comments. One way to compact the json would be something like:

"path": ["qc/ITS/ITSTHRTask0/Threshold/Layer0/Stave0/Threshold", "qc/ITS/ITSTHRTask0/Threshold/Layer0/Stave1/Threshold",....],
"names": ["Threshold_l0_0", "Threshold_l0_1", ....] 

And the same for the dead pixels. This for me would be (probably) the easiest since our real paths on CCDB are the one indicated into "paths" while the "names" are just names to distinguish the different branches of the tree and are not part of the path. In fact I do retrieveMO(dataSource.path, "") to retrieve the object and not retrieveMO(dataSource.path, dataSource.name). If you agree, I propose to go ahead with the PR and then I will do this modification asap (it will take some time because I will have also to modify the TrendingTaskConfig). Please let me know your opinion. Thanks a lot.

Reply:
or as you said but:
"path": "qc/ITS",
"names": ["ITSTHRTask0/Threshold/Layer0/Stave0/Threshold", ....]

but in this case I would have very long names of the branches (provided that one doesn't not any string editing).

Reply:
As you prefer. though I would prefer if the original TrendingTaskConfig was kept as it is. Longer branch names are not a big deal IMO, but probably I am looking at it from a performance point of view, not user convenience :)
Anyway, please go forward as you and Barth like, I will have a closer look after I am back in August.

Reply:
Thanks a lot @knopers8, I will work on this soon.